src/ScoreCard.js

```diff
@@ -0,0 +1,81 @@
+function ScoreCard() {
+  this.score = 0;
+  this.currentFrame = 1;
+  this.totalFrames = 10;
+  this.maxFrames = 12;
+  this.currentBowl = 1;
+  this.isGameOver = false;
+  this.frames = [];
+  for (frameIndex=0; frameIndex <= this.maxFrames; frameIndex +=1 ) this.frames[frameIndex] = {
+    bowl1: 0, 
+    bowl2: 0, 
+    score: 0,
+    strike: false,
+    spare: false,
+  }
+};
+
+ScoreCard.prototype.hitPins = function(numberOfPins){
+  if (this.currentBowl === 1) {
```

Comment:
Try to extract conditional checks into their own, descriptive method (ok, this one is not the best example as it's quite clear, but it's a good habit to get into)